From 63d246625235475ad3337f343a6c491e4021f318 Mon Sep 17 00:00:00 2001
From: John Bresnahan <jbresnah@redhat.com>
Date: Wed, 18 Sep 2013 19:18:43 -1000
Subject: [PATCH] Remove runtime dep on python pbr

---
 glance/version.py | 29 +++++++++++++++++++++++++++--
 1 file changed, 27 insertions(+), 2 deletions(-)

diff --git a/glance/version.py b/glance/version.py
index 0172206..ac67472 100644
--- a/glance/version.py
+++ b/glance/version.py
@@ -13,6 +13,31 @@
 #    under the License.
 
 
-import pbr.version
+GLANCE_VENDOR = "OpenStack Foundation"
+GLANCE_PRODUCT = "OpenStack Glance"
+GLANCE_PACKAGE = None  # OS distro package version suffix
 
-version_info = pbr.version.VersionInfo('glance')
+loaded = False
+
+
+class VersionInfo(object):
+    release = "REDHATGLANCERELEASE"
+    version = "REDHATGLANCEVERSION"
+
+    def version_string(self):
+        return self.version
+
+    def cached_version_string(self):
+        return self.version
+
+    def release_string(self):
+        return self.release
+
+    def canonical_version_string(self):
+        return self.version
+
+    def version_string_with_vcs(self):
+        return self.release
+
+
+version_info = VersionInfo()
